agent/Cargo.toml

```diff
@@ -61,10 +61,10 @@ tempfile = "3.1.0"
 
 [features]
 # To embed discovery handlers, add the desired discovery handler features to default and "agent-full".
-# Or set the features when building the agent with  
+# Or set the features when building the agent with
 # `cargo build --manifest-path agent/Cargo.toml --features "agent-full onvif-feat opcua-feat udev-feat"`
 default = []
 onvif-feat = [ "akri-onvif"]
 opcua-feat = ["akri-opcua"]
 udev-feat = ["akri-udev"]
-agent-full = []
\ No newline at end of file
+agent-full = []
```

Comment:
keep agent-full out by default?

Reply:
i believe that this is just a spacing fix. it is still as it was set

Reply:
ahh got it